appsec/src/helper/client.hpp

```diff
@@ -66,7 +66,7 @@ class client {
 protected:
     template <typename T>
     std::shared_ptr<typename T::response> publish(
-        typename T::request &command, bool rasp = false);
+        typename T::request &command, const std::string &rasp_rule = "");
```

Comment:
Eventually you'll have RASP rule_variant as well, you could just pass a struct such as "rasp_context" containing everything needed (so you can also extend it)

Reply:
Let's wait until then